src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -99,10 +105,45 @@ const LightUpChinatownPage = () => {
         </CostBreakdownImageContainer>
       </DonationContainer>
       <LightUpFaq />
+      <Banner>
+        <PartnerThanksTitle>
+          {t('lightUpChinatown.partnerThanks')}
+        </PartnerThanksTitle>
+        <PartnersLogoContainer>
+          <PartnerLogo src={partners_46} alt="Logo" />
+          <PartnerLogo src={partners_ccba} alt="Logo" />
+          <PartnerLogo src={partners_cccny} alt="Logo" />
+          <PartnerLogo src={partners_prm} alt="Logo" />
+          <PartnerLogo src={partners_udooda} alt="Logo" />
```

Comment:
a more descriptive alt would be great, for usability reasons (not that our website is super optimized for the blind yet, but baby steps)